django/contrib/admin/static/admin/css/rtl.css

```diff
@@ -235,19 +235,19 @@ fieldset .fieldBox {
   background-position: 0 -112px;
 }
 
-a.selector-chooseall {
+button.selector-chooseall {
     background: url(../img/selector-icons.svg) right -128px no-repeat;
 }
 
-a.active.selector-chooseall:focus, a.active.selector-chooseall:hover {
+button.active.selector-chooseall:focus, button.active.selector-chooseall:hover {
     background-position: 100% -144px;
 }
 
-a.selector-clearall {
+button.selector-clearall {
     background: url(../img/selector-icons.svg) 0 -160px no-repeat;
 }
 
-a.active.selector-clearall:focus, a.active.selector-clearall:hover {
+button.active.selector-clearall:focus, button.active.selector-clearall:hover {
```

Comment:
I kind of wonder if the `button` part is stopping something from breaking or if it's just extraneous. I know I've been trying to avoid using elements as part of the selector where possible.